modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -689,7 +691,8 @@ export default {
         menu.push({
           label: {
             key: 'apostrophe:takeActionAndCreateNew',
-            saveLabel: this.$t(this.saveLabel)
+            saveLabel: this.$t(this.saveLabel),
+            typeLabel
```

Comment:
```suggestion
            saveLabel: this.$t(this.saveLabel)
```

there is no  `typeLabel` used in the `apostrophe:takeActionAndCreateNew` key, I think that we can remove it